_monitoring-plugins/alerting/api.md

```diff
@@ -514,12 +514,12 @@ POST _plugins/_alerting/monitors
 Introduced 1.0
 {: .label .label-purple }
 
-When you update a monitor, include the current version number as a parameter. OpenSearch increments the version number automatically (see the sample response).
+When you update a monitor, you can use the `seq_no` and `primary_term` parameters to update an existing policy with optimistic lock. If these numbers don’t match the existing policy or the policy doesn’t exist, Alerting throws an error. OpenSearch increments the version number and the sequence number automatically (see the sample response).
```

Comment:
Can you provide a bit more context into what an optimistic lock is? I think explicitly stating what it is would help with reader comprehension.

Also, I would go with "...the alerting plugin throws an error" instead of "...Alerting throws an error".

Reply:
> Can you provide a bit more context into what an optimistic lock is? I think explicitly stating what it is would help with reader comprehension.

Sorry for using wrong term. optimistic lock should be replaced to optimistic concurrency control. 
https://www.elastic.co/guide/en/elasticsearch/reference/current/optimistic-concurrency-control.html

> Also, I would go with "...the alerting plugin throws an error" instead of "...Alerting throws an error".
Okay. Let me fix it.